.github/workflows/node.js-tests.yml

```
@@ -58,12 +58,17 @@ jobs:
           echo 'SHOW_NEW_CURRICULUM=true' >> .env
           cat .env
 
+      - name: Install node_modules
+        run: pnpm install
+
+      - name: Check formatting
+        run: pnpm prettier --check .
```

```suggestion
        run: pnpm prettier --check . || [ $? -eq 1 ] && echo "\n\nTip: Run 'pnpm run format' in your terminal to fix this.\n\n"
```

---

```suggestion
        run: pnpm prettier --check . || printf "\nTip: Run 'pnpm run format' in your terminal to fix this.\n\n"
```
echo doesn't always print newlines and `||` will already only run the RHS if the LHS returns an error code.

---

Fine with the newlines, I default zsh everywhere, so I would not know.

> ... `||` will already only run the RHS if the LHS returns an error code.

Trying to be smart, are we? We are specifically looking for error code 1. Not 127, not anything else.

Anyways, add a message here in the run-script layer or wherever - All I know is that `--check` is not doing too much than before.

I'll get around to adding GitHub annotations when I get a chance. For now, I'll let you choose whatever you like.

---

If, somehow, the error code ends up being something other than 1 or 0, we can fix that when it happens. If ain't broke :shrug:

---

> If, somehow, the error code ends up being something other than 1

In that case ignore this request and just go back to `pnpm prettier --check .`. 

Because this is NOT a case of if ain't broke. The error codes are a thing, and they matter. The message should be displayed in a specific case, on a signal specifically emitted by the command. 

If we are casual about it OR don't know how it works, then we might not even do a half-measure by adding a message. The original change is just fine.

---

I don't think this deserves more discussion, but I'm fine with the error code = 1 check, so I added it back.